demo/agilebank/constraints/probes_must_be_provided.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sRequiredProbes
+metadata:
+  name: must-have-probes
+spec:
+  match:
+    kinds:
+      - apiGroups: ["apps", ""]
```

Comment:
i believe we only need `extensions` (deprecated) and `apps` here

Reply:
Thanks @sozercan. What about Pods? Should it be `""` or `"core"`? Right now I have `""` to cover the `Pod` kind and it's working.

Reply:
for pods, we should have `""`